website/static/js/koHelpers.js

```diff
@@ -44,6 +44,15 @@ var makeRegexValidator = function(regex, message, match) {
     };
 };
 
+var cleanURL = function(value) {
```

Comment:
Is it possible to use the existing `ensureHttp` validator instead of creating a new one? https://github.com/billyhunt/osf.io/blob/feature/multiple-profile-websites2/website/static/js/koHelpers.js#L60-#L65